tensorflow/compiler/xla/stream_executor/gpu/gpu_blas_lt.cc

```diff
@@ -31,6 +31,16 @@ using blas::ComputationType;
 using blas::DataType;
 using xla::PrimitiveType;
 
+bool GpuBlasLtEnabled() {
```

Comment:
huh I remember in gemm_rewriter.cc I use xla_gpu_enable_cublaslt flag to enable hipblaslt. 
I think we should do it here as well to have just 1 global switch for hipblaslt. 
Can you replace this function with this one please:
```
bool GpuBlasLtEnabled() {
  static std::atomic_bool result{[] {
    return xla::GetDebugOptionsFromFlags().xla_gpu_enable_cublaslt();
  }()};
  return result;
}
```

Reply:
updated